tools/public_api_guard/generate-guard-tests.bzl

```diff
@@ -7,15 +7,19 @@ load("@npm//ts-api-guardian:index.bzl", "ts_api_guardian_test")
   against the associated source entry point.
 """
 
-def generate_test_targets(golden_files):
-    for golden_file in golden_files:
+def generate_test_targets(targets):
```

Comment:
I was split between doing this here or when generating the target names in `tools/public_api_guard/BUILD.bazel`. I decided to do it here, because even though it would be cleaner when generating the target names, I think that the slashes in the target name make sense, they just produce a weird directory structure when the golden is generated.